engine/Default/galactic_post_past.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require_once(get_file_loc('menu.inc'));
+create_galactic_post_menu();
+$template->assign('PageTopic', 'Past <i>Galactic Post</i> Editions');
+
+// View past editions of current game by default
+if (isset($_POST['game_id'])) {
+	SmrSession::updateVar('game_id', $_POST['game_id']);
+} else if (!isset($var['game_id'])) {
+	SmrSession::updateVar('game_id', $player->getGameID());
+}
+$template->assign('SelectedGame', $var['game_id']);
+
+// Get the list of games with published papers
+// Add the current game to this list no matter what
+$db->query('SELECT game_name, game_id FROM game WHERE game_id IN (SELECT DISTINCT(game_id) FROM galactic_post_paper WHERE online_since IS NOT NULL) OR game_id=' . $db->escapeNumber($player->getGameID()) . ' ORDER BY game_id DESC');
+$publishedGames = array();
+while ($db->nextRecord()) {
+	$publishedGames[] = array('game_name' => $db->getField('game_name'),
```

Comment:
It might be cool if we could do something like
```
$publishedGames = $db->map(function ($db) {
	return array('game_name' => $db->getField('game_name'),
		'game_id' => $db->getInt('game_id'));
});
```
to handle this pattern, although it's not a huge difference in terms of verbosity